app/Providers/ModelServiceProvider.php

```diff
@@ -53,9 +54,21 @@ public function boot()
         // When Reviews are saved create an event for them.
         Review::saved(function ($review) {
             info('review', ['campaign' => $review->post->campaign_id]);
+
             // Update the "counter cache" on this campaign:
             $review->post->campaign->refreshCounts();
 
+            // If this post is associated with a school, update the school action stats.
+            $schoolId = $review->post->school_id;
+
+            if ($schoolId) {
+                $actionId = $review->post->action_id;
+                ActionStat::updateOrCreate(
+                    ['action_id' => $actionId, 'school_id' => $schoolId],
+                    ['accepted_quantity' => Post::getAcceptedQuantitySum($actionId, $schoolId)]
```

Comment:
Testing this a little more throughly (having worked fixing inbox locally over in #966) - when I debug the `$review->post` here, its status is still `pending`, so first review results in a quantity of 0.  Will update this branch with the fix, but I think the campaign refreshCounts should get moved to the same place? Seems it: https://www.pivotaltracker.com/n/projects/2328687/stories/170111820